src/clientSideScene/sceneInfra.ts

```diff
@@ -27,7 +27,6 @@ import {
   Y_AXIS,
```

Comment:
Many files will have their interface reference swapped.

From `import type { EngineCommandManager } from '@src/lang/std/engineConnection'` to `import type { ConnectionManager } from '@src/network/connectionManager'`

the interface is roughly the same. Anything that is different I've been handling.